packages/next/build/index.ts

```diff
@@ -455,13 +462,31 @@ export default async function build(dir: string, conf = null): Promise<void> {
     await fsRmdir(exportOptions.outdir)
     await fsWriteFile(manifestPath, JSON.stringify(pagesManifest), 'utf8')
   }
+
+  if (sprPages.size > 0) {
+    const prerenderRoutes: any[] = []
```

Comment:
nit: seems pretty arbitrary that this is `any`, considering we know the type :P

Reply:
Added a type